ts/store/middlewares/analytics.ts

```diff
@@ -122,47 +119,55 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
         mp.track(action.type).then(constNull, constNull),
         identify.then(constNull, constNull)
       ]);
-
+    //
+    // Authentication login actions (with properties)
```

Comment:
what's the difference between `Authentication actions` and `Authentication login actions` ?